### PR TITLE
added a comment to experiment

### PR DESCRIPTION
**Pull request by ponjae**
I want you to look a these changes

### CODE REVIEW

File: lab.java [hunk 1]
```diff
@@ -12,7 +12,7 @@ public static void main(String[] args) {
         String wordToSearchFor = inputs[1];
         String filename = inputs[2];
         //test
```
Comment (OscarWiklund96) [inline lab.java:14]: inputs[0] is never used?

Comment (ponjae) [reply lab.java:14]: Yes your are right, if this would be a proper program I would change it but now I don't care really..

**Review by OscarWiklund96 — CHANGES_REQUESTED** [review]
I think the new comment is unneccesary as it does not describe anythign related to the program.

**Review by OscarWiklund96 — APPROVED** [review]
Looks better

**Pull request merged**